app/src/main/java/org/schabi/newpipe/about/AboutActivity.java

```diff
@@ -195,13 +209,21 @@ public int getCount() {
 
         @Override
         public CharSequence getPageTitle(final int position) {
-            switch (position) {
-                case 0:
-                    return getString(R.string.tab_about);
-                case 1:
-                    return getString(R.string.tab_licenses);
+            int newPosition = getLayoutPosition(position);
+
+            if (newPosition == 0) {
+                return getString(R.string.tab_about);
+            } else if (newPosition == 1) {
+                return getString(R.string.tab_licenses);
             }
+
             return null;
         }
+
+        private int getLayoutPosition(final int position) {
```

Comment:
As this method is duplicated thrice, should make it static util function, `int getLayoutPosition(boolean isRTL, int count, int position)`